L1Trigger/Phase2L1ParticleFlow/src/JetId.cc

```diff
@@ -19,8 +40,9 @@ BJetId::BJetId(const std::string &iInput, const std::string &iOutput, const BJet
   fOutput_ = iOutput;
 }
 
-BJetId::~BJetId() {}
-void BJetId::setNNVectorVar() {
+JetId::~JetId() {}
```

Comment:
Could you remove this and set the destructor to default in the class specification? After that is done, I think that everything seems in order here at least from me. @epalencia any objection to merging this into the prototype? If not @ddiaz006 I believe there is already a PR open for this in CMSSW, could you just make sure it is up to date with any changes you have made in review here?